src/Nethermind/Nethermind.Db.Rocks/DbOnTheRocks.cs

```diff
@@ -501,6 +505,7 @@ private static WriteOptions CreateWriteOptions(PerTableDbConfig dbConfig)
                 }
             }
 
+            using var handle = _readThrottle.Acquire();
             return _db.Get(key, cf);
```

Comment:
Doesnt this mean it can't do concurrent read anymore?

Reply:
Removed in https://github.com/NethermindEth/nethermind/pull/6438 as wasn't sure about this.